src/data/grammar/grammar.json

```diff
@@ -16,6 +13,13 @@
         "#inGraphics#",
         "#headlineKeepOrMulligan#"
     ],
+    "headline": [
+        "Wizards Announces: #novelCardSetName.capitalizeAll#",
+        "[_nextSet:The next set,The next fall set,#codename.capitalize#][_verbphrase:is actually,is really named,will be named]#_nextSet# #_verbphrase# #novelCardSetName.capitalizeAll#"
+    ],
+    "inGraphics": [
```

Comment:
I think in the latest, `inGraphics` is in `mtg-in-graphics` now so it shouldn't be in `grammar`? Confirm in your local copy post-merge from master, but I think this shouldn't be here now.